core/controlplane/config/templates/cloud-config-worker

```diff
@@ -142,7 +142,8 @@ coreos:
         {{end}}--register-node=true \
         {{if .Experimental.Taints}}--register-schedulable=false \
         {{end}}--allow-privileged=true \
-        --pod-manifest-path=/etc/kubernetes/manifests \
+        {{if .Experimental.NodeStatusUpdateFrequency}}--node-status-update-frequency={{.Experimental.NodeStatusUpdateFrequency}}
```

Comment:
@mumoshu So I attempted to add this setting, however, the setting does not seem to get populated during the rendering of the worker template.  How do I get this setting to be available in rendering the worker's user-data?  I think I'm missing something about how these settings get into the nodepools.  Thanks!  As is, this setting does not get applied when set.